data/README.md

```diff
@@ -39,18 +39,27 @@ instance, they should be in the same format as the files below.
   Contains signature (node) activity levels (as absolute values) for each
   sample in the compendium derived from the eADAGE machine learning model.
   In this file, samples are rows, and signatures are columns. This file is
-  based on the
-  `all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt`
-  file, which is now in the `data/old/` folder and was generated by
+  based on:
+  ```
+  all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt
```

Comment:
the main reason why I changed this line was because originally the super-long file name and description text were mingled together and seemed a little messy.

Reply:
Sure, I agree that this looks better!